source/converter.py

```diff
@@ -0,0 +1,157 @@
+tips = '''
+Another tip is only use either "" or '' dont switch between them
+
+Also use f"" to print strings with variables in them simply put every variable in a bracket
+EXAMPLE: print(f"Hello i am {name}")
+
+When naming variables use names that descripe the value of the variable not random letters
+EXAMPLE: windows_path = 'C:/Windows'
+'''
+import ctypes
+import os, sys
+import time
+import requests
+from pytube import YouTube
+
+banner = '''
+
+███╗   ███╗██╗   ██╗██╗  ████████╗██╗██████╗  ██████╗ ██╗    ██╗███╗   ██╗██╗      ██████╗  █████╗ ██████╗ 
+████╗ ████║██║   ██║██║  ╚══██╔══╝██║██╔══██╗██╔═══██╗██║    ██║████╗  ██║██║     ██╔═══██╗██╔══██╗██╔══██╗
+██╔████╔██║██║   ██║██║     ██║   ██║██║  ██║██║   ██║██║ █╗ ██║██╔██╗ ██║██║     ██║   ██║███████║██║  ██║
+██║╚██╔╝██║██║   ██║██║     ██║   ██║██║  ██║██║   ██║██║███╗██║██║╚██╗██║██║     ██║   ██║██╔══██║██║  ██║
+██║ ╚═╝ ██║╚██████╔╝███████╗██║   ██║██████╔╝╚██████╔╝╚███╔███╔╝██║ ╚████║███████╗╚██████╔╝██║  ██║██████╔╝
+╚═╝     ╚═╝ ╚═════╝ ╚══════╝╚═╝   ╚═╝╚═════╝  ╚═════╝  ╚══╝╚══╝ ╚═╝  ╚═══╝╚══════╝ ╚═════╝ ╚═╝  ╚═╝╚═════╝ 
+
+                        * Open Sorce Multi Downloader by CraftModzZ aka. WeepingAngel *
+                Discord: _WeepingAngel_ VI#6666 | GitHub: https://www.github.com/crafttino21
+
+
+'''
+
+
+class colors:
+  '''
+  This is a class containing variables of similar values
+  To use this class call the class name then the variable 
+  '''
+  blue = os.system("color b")
+  red = os.system("color C")
+  green = os.system("color a")
+  pink = os.system("color D")
+
+

```

Comment:
This is called a doc-string and its for displaying informatation about the function or class in plain text.